docs/OV_Runtime_UG/Integrate_with_customer_application.md

```diff
@@ -66,235 +61,124 @@ Use the following code to create Inference Engine Core to manage available devic
 
-   @snippet snippets/Integrate_with_customer_application_new_API.cpp part9
+   @snippet snippets/src/main.cpp part8
 
-* **Optimal way to handle ROI (a ROI object located inside of input of one network is input for another).** It is possible to re-use shared input by several networks. You do not need to allocate separate input blob for a network if it processes a ROI object located inside of already allocated input of a previous network. For instance, when first network detects objects on a video frame (stored as input blob) and second network accepts detected bounding boxes (ROI inside of the frame) as input. In this case, it is allowed to re-use pre-allocated input blob (used by first network) by second network and just crop ROI without allocation of new memory using `InferenceEngine::make_shared_blob()` with passing of `InferenceEngine::Blob::Ptr` and `InferenceEngine::ROI` as parameters.
+* **Optimal way to handle ROI (a ROI object located inside of input of one network is input for another).** It is possible to re-use shared input by several networks. You do not need to allocate separate input blob for a network if it processes a ROI object located inside of already allocated input of a previous network. For instance, when first network detects objects on a video frame (stored as input blob) and second network accepts detected bounding boxes (ROI inside of the frame) as input. In this case, it is allowed to re-use pre-allocated input blob (used by first network) by second network and just crop ROI without allocation of new memory using `ov::Tensor()` with passing of `ov::Tensor` and `ov::Coordinate` as parameters.
 
-   @snippet snippets/Integrate_with_customer_application_new_API.cpp part10
+   @snippet snippets/src/main.cpp part9
 
    Make sure that shared input is kept valid during execution of each network. Otherwise, ROI blob may be corrupted if the original input blob (that ROI is cropped from) has already been rewritten.
 
-* Allocate input blobs of the appropriate types and sizes, feed an image and the input data to the blobs, and call `InferenceEngine::InferRequest::SetBlob()` to set these blobs for an infer request:
+* Allocate input blobs of the appropriate types and sizes, feed an image and the input data to the blobs, and call `ov::InferRequest::set_tensor()` to set these tensors for an infer request:
 
-   @snippet snippets/Integrate_with_customer_application_new_API.cpp part11
+   @snippet snippets/src/main.cpp part10
 
-A blob can be filled before and after `SetBlob()`.
+A blob can be filled before and after `set_tensor()`.
 
 > **NOTE**:
 >
-> * The `SetBlob()` method compares precision and layout of an input blob with the ones defined in step 3 and
+> * The `set_tensor()` method compares precision and layout of an input blob with the ones defined in step 3 and
```

Comment:
we don't have layout for the tensor now. It compares shapes and element type
precision should be removed with element type in the document

Reply:
In general, this "note" is obsolete and we need to remove CV / G-API semantic